templates/projects/list.tmpl

```diff
@@ -51,7 +51,7 @@
 							{{JsPrettyNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
 						</span>
 					</div>
-					{{if and (or $.CanWriteIssues $.CanWritePulls) (not $.Repository.IsArchived)}}
+					{{if and $.CanWriteProjects (not $.Repository.IsArchived)}}
 					<div class="ui right operate">
 						<a href="{{$.Link}}/{{.ID}}/edit" data-id={{.ID}} data-title={{.Title}}>{{svg "octicon-pencil"}} {{$.locale.Tr "repo.issues.label_edit"}}</a>
 						{{if .IsClosed}}
```

Comment:
By the way, we still have the `$.Link` problems below this line.

Reply:
`{{$.Link}}/{{.ID}}` is correct here, but seems strange.
And I forgot to fix  `#62`, we do it in this PR.

Reply:
`$.Link` is defined in `#688`:
https://github.com/go-gitea/gitea/blob/c775053476295c876791889030fb734dca5c0783/modules/context/context.go#L685-L690